lib/gs-app-list-private.h

```diff
@@ -65,7 +65,6 @@ GsAppList	*gs_app_list_copy		(GsAppList	*list);
 guint		 gs_app_list_get_size_peak	(GsAppList	*list);
 void		 gs_app_list_filter_duplicates	(GsAppList	*list,
 						 GsAppListFilterFlags flags);
-void		 gs_app_list_randomize		(GsAppList	*list);
```

Comment:
Not sure you need to move this to the public header actually... `src/gs-discovery-feed-content-provider.c` already includes `lib/gs-app-list-private.h` as far as I can see.

Reply:
Yeah, I saw that now actually. I'll remove the commit, looks like this is not necessary \o/